boefjes/boefjes/katalogus/dependencies/plugins.py

```diff
@@ -73,6 +74,9 @@ def by_plugin_id(self, plugin_id: str, organisation_id: str) -> PluginType:
 
         raise KeyError(f"Plugin {plugin_id} not found for {organisation_id}")
 
+    def by_plugin_ids(self, plugin_ids: list[str], organisation_id: str) -> list[PluginType]:
+        return [self.by_plugin_id(plugin_id, organisation_id) for plugin_id in plugin_ids]
```

Comment:
This might be a performance drawback, since this does repeated calls to the plugin service's `by_plugin_id`, which in turn makes calls to the repository storage's `get_all`

Reply:
You might get a similar effect as this one https://github.com/minvws/nl-kat-coordination/issues/2524, although that's more related to the plugin states

Reply:
Its alot quicker now.